nix/nixos/tx-generator-service.nix

```diff
@@ -1,5 +1,7 @@
 pkgs:
 let
+  plutus = true;
+  plutusScriptFile = "../../../bench/script/sum1ToN.plutus";
```

Comment:
Here the Plutus settings are hardcoded.
Instead proper parameters from the nix-service definition should be used instead.